modules/friendlytalkback.js

```diff
@@ -338,6 +338,11 @@ Twinkle.talkback.noticeboards = {
 		label: 'WP:OTRS/N (OTRS noticeboard)',
 		text: '{{OTRSreply|1=$SECTION|2=~~~~}}',
 		editSummary: 'You have replies at the [[Wikipedia:OTRS noticeboard|OTRS noticeboard]]'
+	},
+	'effp': {
+		label: 'WP:EFFP/R (Edit filter false positive report)',
+		text: '{{EFFPReply|1=$SECTION|2=~~~~}}',
+		editSummary: 'You have replies at the [[WP:EFFP/R|Edit filters false positive report]] page'
```

Comment:
```suggestion
		editSummary: 'You have replies at the [[Wikipedia:Edit filter/False positives/Reports|Edit filters false positive report]] page'
```

Might as well use the full link.
----
It'd be nonstandard, but is there a reason not to say something like "You have replies to your edit filter false positive report"?  It's already somewhat nonstandard, and makes a tad more sense.  Regardless, the language around this feels off.